tests/test_registration.py

```diff
@@ -1,5 +1,7 @@
-from pages import OTPPage, AddComplaintPage
+from pages import OTPPage, AddComplaintPage, ComplaintTypePage, TopMenuPage, BottomMenuPage, ProfilePage
```

Comment:
Import every page in new line so that conflicts can be resolved easily when merging